project_agile/views/project_project_views.xml

```diff
@@ -0,0 +1,233 @@
+            </div>
+
+             <field name='user_id' position="after">
+                 <field name="agile_enabled"/>
+                 <field name="agile_method"
+                        attrs="{'invisible': [('agile_enabled', '=', False)]}"
+                        options="{'horizontal': true}"
+                        widget="radio"/>
+             </field>
+        </field>
+    </record>
+
+    <!-- Project Search View  -->
+    <record id="view_project_project_filter_extend_with_key" model="ir.ui.view">
+        <field name="name">project.type.search</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_project_filter"/>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="type_id" string="Project Type"/>
+            </field>
+
+            <group position="inside">
+                <filter string="Type" name="Type" context="{'group_by':'type_id'}"/>
+                <filter string="Agile" name="Agile" context="{'group_by':'agile_enabled'}"/>
+                <filter string="Agile Method" name="Agile Method" context="{'group_by':'agile_method'}"/>
+            </group>
+        </field>
+    </record>
+
+
+
+    <!-- Dashboard: Project Kanban -->
+    <record model="ir.ui.view" id="view_project_kanban">
+        <field name="name">project.project.kanban</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project.view_project_kanban"/>
+        <field name="priority" eval="100"/>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="type_id"/>
+            </field>
+
+            <xpath expr="//div[hasclass('o_primary')]/span[1]" position="after">
+                <span t-if="record.type_id.value">
+                    <t t-esc="record.type_id.value"/>
+                </span>
+
+            </xpath>
+
+
+            <xpath expr="//div[hasclass('o_kanban_manage_reports')]/div[2]" position="before">
+                <div>
+                    <a name="open_epics" type="object">Epics</a>
+                </div>
+                <div>
+                    <a name="open_user_stories" type="object">User Stories</a>
+                </div>
+
+            </xpath>
+
+            <xpath expr="//a[text()='Tasks']" position="attributes">
+                <attribute name="name">open_tasks</attribute>
+                <attribute name="type">object</attribute>
+            </xpath>
+
+            <xpath expr="//t[@t-esc='record.task_count.value']/../../.." position="attributes">
+                <attribute name="name">open_tasks</attribute>
+                <attribute name="type">object</attribute>
+            </xpath>
+
+            <xpath expr="//a[@t-if='record.task_needaction_count.raw_value']" position="attributes">
```

Comment:
This has changed as well: https://github.com/odoo/odoo/commit/8438470c3a578a2b0f4f4187936466a754e601d7#diff-5a45d5b004d555a5805f0892644b41b7L332